docs/content.zh/release-notes/flink-1.14.md

```diff
@@ -0,0 +1,424 @@
+filesystem connector with an ORC format as a replacement. It is possible to read via Table & SQL API
+and convert the Table to DataStream API if necessary. DataSet API is not supported anymore.
+
+#### Drop usages of BatchTableEnvironment and old planner in Python
+
+##### [FLINK-22619](https://issues.apache.org/jira/browse/FLINK-22619)
+
+The Python API does not offer a dedicated BatchTableEnvironment anymore. Instead, users can switch
+to the unified TableEnvironment for both batch and stream processing. Only the Blink planner (the
+only remaining planner in 1.14) is supported.
+
+#### Migrate ModuleFactory to the new factory stack
+
+##### [FLINK-23720](https://issues.apache.org/jira/browse/FLINK-23720)
+
+The `LOAD/UNLOAD MODULE` architecture for table modules has been updated to the new factory stack of
+FLIP-95. Users of this feature should update their `ModuleFactory` implementations.
+
+#### Migrate Table API to new KafkaSink
+
+##### [FLINK-23639](https://issues.apache.org/jira/browse/FLINK-23639)
+
+Table API/SQL write to Kafka with the new KafkaSink.
+
+### Connectors
+
+#### Implement FLIP-179: Expose Standardized Operator Metrics
+
+##### [FLINK-23652](https://issues.apache.org/jira/browse/FLINK-23652)
+
+Connectors using unified Source and Sink interface expose certain standardized metrics
+automatically.
+
+#### Port KafkaSink to FLIP-143
+
+##### [FLINK-22902](https://issues.apache.org/jira/browse/FLINK-22902)
+
+KafkaSink supersedes FlinkKafkaProducer and provides efficient exactly-once and at-least-once
+writing with the new unified sink interface supporting both batch and streaming mode of DataStream
+API.
+
+#### Deprecate FlinkKafkaConsumer
+
+##### [FLINK-24055](https://issues.apache.org/jira/browse/FLINK-24055)
+
+FlinkKafkaConsumer has been deprecated in favor of KafkaSource, FlinkKafkaProducer has been
+deprecated in favor of KafkaSink.
+
+#### Pulsar Source
+
+##### [FLINK-20731](https://issues.apache.org/jira/browse/FLINK-20731)
+
+Flink now directly provides a way to read data from Pulsar with DataStream API.
+
+#### Add Support for Azure Data Lake Store Gen 2 in Flink File System
+
+##### [FLINK-18562](https://issues.apache.org/jira/browse/FLINK-18562)
+
+Flink now supports reading from Azure Data Lake Store Gen 2 with the flink-azure-fs-hadoop
+filesystem using the abfs(s) scheme.
+
+#### InputStatus should not contain END_OF_RECOVERY
+
+##### [FLINK-23474](https://issues.apache.org/jira/browse/FLINK-23474)
+
+`InputStatus.END_OF_RECOVERY` was removed. It was an internal flag that should never be returned from
+SourceReaders. Returning that value in earlier versions might lead to misbehaviour.
+
+#### Connector-base exposes dependency to flink-core.
+
+##### [FLINK-22964](https://issues.apache.org/jira/browse/FLINK-22964)
+
+Connectors do not transitively hold a reference to `flink-core` anymore. That means that a fat jar
+with a connector does not include `flink-core` with this fix.
+
+### Runtime & Coordination
```

Comment:
@dawidwys Sorry for the late updates. Would you like to include the release notes of FLINK-21924 into the "Runtime & Coordination" section?

Reply:
Isn't it just an ad for the feature? How does it match the purpose of the page?

> These release notes discuss important aspects, such as configuration, behavior, or dependencies, that changed between
Flink 1.x and Flink 1.y. Please read these notes carefully if you are planning to upgrade your Flink version to 1.y.

Doesn't it fit this page more? https://github.com/apache/flink-web/pull/466

Reply:
I just see the Pulsar connector, which is also a new feature, was added. Sorry for the misunderstanding. If the release note just contains capability changes, FLINK-21924 should not be included here.

Reply:
I agree it's not consistent. I feel there is a misunderstanding/there is no common agreement on what to put in here. I tried removing just new features if I spotted them.

Reply:
btw, I sent a short note to the ML about it: https://lists.apache.org/thread.html/r675956b2d333a9b1c73e3c3b942dd25bcbb12903f00ccfdda9f3610b%40%3Cdev.flink.apache.org%3E